Sources/Schema.swift

```diff
@@ -100,7 +101,27 @@ extension SchemaBuilder {
         }
 
         let name = components[2]
+
+        if let cachedSchema = ReferenceResolver.cachedSchemaReferences[name] {
+            return Structure(name: name, structure: cachedSchema)
+        }
+        if let resolving = ReferenceResolver.resolvingReferences[name], resolving {
+            return Structure(name: name, structure: .unresolvedReference(pointer.path))
```

Comment:
If it threw an error here other references in the spec wouldn't have a chance to resolve and then cache themselves, that's why this intermediate step is needed